addon/ng2/commands/install.js

```diff
@@ -0,0 +1,39 @@
+/* jshint node: true, esnext: true */
+'use strict';
+
+const Command      = require('ember-cli/lib/models/command');
+const SilentError  = require('silent-error');
+const Promise      = require('ember-cli/lib/ext/promise');
+const InstallTask  = require('../tasks/install');
+
+module.exports = Command.extend({
+  name: 'install',
+  description: 'Adds 3rd party library to existing project',
+  works: 'insideProject',
+
+  availableOptions: [
+    { name: 'typings', type: String, aliases: ['t'], description: 'Installs specified typings' }
```

Comment:
I'm not sure we need this. We should check if a typings exist for a node package we're trying to install. Is there a case where this doesn't work?